biome.json

```diff
@@ -20,6 +20,9 @@
       "suspicious": {
         "all": false,
         "noControlCharactersInRegex": "error"
+      },
+      "nursery": {
+        "noUnusedImports": "error"
```

Comment:
Can you change package.json biome `fix` command to `--apply-unsafe` to `--apply` so we don't have any breaking changes?

Reply:
Done with f4b116978f7ce95764df30ef03149f4c6c626e11